### PR TITLE
[AzurePowerShellV2 AzurePowerShellV3 AzurePowerShellV4 AzurePowerShellV5] Added telemetry tracking

### PR DESCRIPTION
**Pull request by vsafonkin**
**Task name**: AzurePowerShellV2 AzurePowerShellV3 AzurePowerShellV4 AzurePowerShellV5

**Description**: added telemetry tracking

**Documentation changes required:** (Y/N) no

**Added unit tests:** (Y/N) no

**Attached related issue:** (Y/N) [#1406](https://github.com/actions/virtual-environments-internal/issues/1406)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzurePowerShellV3/AzurePowerShell.ps1 [hunk 1]
```diff
@@ -180,3 +180,6 @@ finally {
 Write-Host "## Script Execution Complete"
 Disconnect-AzureAndClearContext -authScheme $authScheme -ErrorAction SilentlyContinue
 
+# Telemetry
+$telemetryJsonContent = "{`"targetAzurePs`":`"$targetAzurePs`"}"
```
Comment (maxim-lobanov) [inline Tasks/AzurePowerShellV3/AzurePowerShell.ps1:184]: May be?
```pwsh
$telemetryJsonContent = @{ targetAzurePs = $targetAzurePs } | ConvertTo-Json
```

Comment (vsafonkin) [reply Tasks/AzurePowerShellV3/AzurePowerShell.ps1]: Done, thank you!

**Review by AlenaSviridenko — APPROVED** [review]

File: Tasks/AzurePowerShellV4/AzurePowerShell.ps1 [hunk 2]
```diff
@@ -263,5 +263,9 @@ finally {
     Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
     Remove-EndpointSecrets
     Disconnect-AzureAndClearContext -restrictContext $restrictContext -ErrorAction SilentlyContinue
+
+    # Telemetry
+    $telemetryJsonContent = @{ targetAzurePs = $targetAzurePs } | ConvertTo-Json
```
Comment (anatolybolshakov) [inline Tasks/AzurePowerShellV4/AzurePowerShell.ps1:268]: Should we apply the same for azurepowershell.ts as well?

Comment (vsafonkin) [reply Tasks/AzurePowerShellV4/AzurePowerShell.ps1:268]: Done

**Comment by anatolybolshakov** [comment]
@AmrutaKawade could you please take a look also?

**Review by maxim-lobanov — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

**Review by egor-bryzgalov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
Tested V1-V5 versions with @maxim-lobanov - works good now.

**Pull request merged**